src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/services.component.html

```diff
@@ -1,11 +1,5 @@
-<cd-alert-panel type="info"
-                *ngIf="!orchestratorExist && !checkingOrchestrator"
-                i18n>Please consult the
-  <a href="{{ docsUrl }}"
-     target="_blank">documentation</a> on how to
-  configure and enable the orchestrator functionality.</cd-alert-panel>
-
-<ng-container *ngIf="orchestratorExist">
+<cd-orchestrator-doc-panel *ngIf="!hasOrchestrator"></cd-orchestrator-doc-panel>
```

Comment:
We should decouple sub-component displaying/hiding from the specific components. In the fix I'm working on for the feature toggles, I'm moving the feature-toggles service into a separate directive, so the host component doesn't need to know about any unrelated services for controlling sub-component visibility, and everything is dealt with at DOM/template level. When extended that for this case, it would work like this:
```ts
<cd-orchestrator-doc-panel *cdDisplayIfNot="'orchestrator'"></cd-orchestrator-doc-panel>
```

And it'd be even easier if we extend the `@component` decorator to accept a `{ display{If,IfNot}: ['<component_label>'] }` metadata and components become displayed/hidden based on that.